neqo-transport/src/cc/classic_cc.rs

```diff
@@ -281,12 +308,22 @@ impl<T: WindowAdjustment> CongestionControl for ClassicCongestionControl<T> {
             &[QlogMetric::BytesInFlight(self.bytes_in_flight)],
         );
 
+        // Lost PMTUD probes do not elicit a congestion control reaction.
+        let lost_packets: Vec<SentPacket> = lost_packets
+            .iter()
+            .filter(|pkt| !self.pmtud.is_pmtud_probe(pkt))
+            .cloned()
+            .collect();
+        if lost_packets.is_empty() {
+            return false;
+        }
+
         let congestion = self.on_congestion_event(lost_packets.last().unwrap());
```

Comment:
```suggestion
        let Some(last_lost_packet) = lost_packets.last() else {
            return false;
        }

        let congestion = self.on_congestion_event(last_lost_packet);
```

Making the invalid state impossible at compile time instead of a panic at runtime.